src/App.js

```diff
@@ -13,27 +16,60 @@ function App() {
           <div className="home">
             <h2 className="home__name">Lions</h2>
 
-            {/* TODO STEP 3 - We need to change the hardcoded values in these divs to accept dynamic values from our state. */}
+            {/* TODO STEP 3 - We need to change the hardcoded values in these divs
+            // to accept dynamic values from our state. */}
 
-            <div className="home__score">32</div>
+            <div className="home__score">{h_value}</div>
           </div>
           <div className="timer">00:03</div>
```

Comment:
You can use moment.js for the timer. Here is a sneak peak at how to do it. https://upmostly.com/tutorials/build-a-react-timer-component-using-hooks